lib/licenses.nix

```diff
@@ -309,6 +309,12 @@ lib.mapAttrs (n: v: v // { shortName = n; }) rec {
     fullName = "GNU General Public License v2.0 only";
   };
 
+  gpl2Classpath = {
+    spdxId = "GPL-2.0 WITH Classpath-exception-2.0";
```

Comment:
Identifier should be GPL-2.0-with-classpath-exception

Reply:
Fixed.